tests/monster_test.fbs

```diff
@@ -84,6 +84,14 @@ table Referrable {
   id:ulong(key, hash:"fnv1a_64");
 }
 
+table NativeInlineTable {
+  a: int;
+}
+
+table TestNativeInlineTable {
+  t: [NativeInlineTable] (native_inline);
+}
+
```

Comment:
Hmm, i wonder if we should just put this in a separate schema file. The reason is, you have all .ts, .py, etc.. changes that are not really relevant, it just because you added two new tables that all the languages are now generating code for them. Which just adds noise.